lib/Entry.hs

```diff
@@ -76,11 +78,18 @@ instance A.FromJSON FieldVisibility where
     "private" -> pure Private
     other -> fail $ "expecting either 'public' or 'private', but found: '" <> T.unpack other <> "'"
 
+newtype FieldValue = FieldValue { fvValue :: T.Text }
+  deriving stock (Show, Eq)
+makeLensesWith abbreviatedFields ''FieldValue
+
+instance Buildable FieldValue where
+  build (FieldValue t) = build t <> build (setSGRCode [Reset])
```

Comment:
Please add an haddock comment to the `Buildable` instance explaining why we're doing this. A link to the #48 issue might also be helpful.

Reply:
Done

Reply:
I see the new comment describes what we're doing, but it doesn't say _why_ (e.g., what issues are we trying to prevent by appending `reset`?).

I think adding a link to #48 would be sufficient - future readers can just go and read the issue to understand why we did this.

Reply:
Done